xrpVendordep/src/main/native/include/frc/xrp/XRPReflectanceSensor.h

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#pragma once
+
+#include <frc/AnalogInput.h>
+
+namespace frc {
+
+/**
+ * This class represents the reflectance sensor pair
+ * on the XRP robot.
+ */
+class XRPReflectanceSensor {
+ public:
+  XRPReflectanceSensor() = default;
+
```

Comment:
```suggestion
```
Looks like we don't even need the default constructor explicitly declared: https://godbolt.org/z/vbKMYdfds